### DISCUSSION TITLE
可以在macos上打包exe吗

### DISCUSSION THREAD

**Discussion by theajack** [tw93/Pake#668] [Q&A]
可以在macos上打包exe吗

**Comment by Tlntin** [comment]
不行，不过可以看wiki，用github action在线打包。

**Reply by wumingluren** [reply]
在线打包能打本地的文件吗？

**Reply by tw93** [reply]
可以的，可见，github actions 的羊毛
https://github.com/tw93/Pake/wiki/%E5%9C%A8%E7%BA%BF%E7%BC%96%E8%AF%91%EF%BC%88%E6%99%AE%E9%80%9A%E7%94%A8%E6%88%B7%E4%BD%BF%E7%94%A8%EF%BC%89
